stubs/defaultConfig.stub.js

```diff
@@ -305,8 +307,19 @@ module.exports = {
       screen: '100vh',
     },
     padding: theme => theme('spacing'),
-    margin: theme => ({ auto: 'auto', ...theme('spacing') }),
-    negativeMargin: theme => theme('spacing'),
+    margin: theme => {
+      const negativeSpacing = Object.keys(theme('spacing'))
+        .filter(key => theme('spacing')[key] !== '0')
+        .reduce((negativeSpacing, key) => {
+          return { ...negativeSpacing, [`-${key}`]: `-${theme('spacing')[key]}` }
```

Comment:
Maybe an implicit return would make the `reduce` a bit nicer?

```suggestion
.reduce((negativeSpacing, key) => ({
  ...negativeSpacing,
  [`-${key}`]: `-${theme('spacing')[key]}`
}))
```

Reply:
Yeah that looks a bit nicer, thanks!